.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
```

Comment:
I'm having trouble following this issue because it seems we have several different discussions going on different topics.   However on the travis issue, it seems like https://github.com/NESII/pyugrid/blob/i96-flexible-mesh/.travis.yml is overly complicated.  

Shouldn't we just install conda, create an environment using `requirements.txt` file and run the test script? I'm thinking like this [simpler example](https://github.com/ioos/odvc/blob/master/.travis.yml#L17)?